app/services/stats/r004_cabinet_office_report.rb

```diff
@@ -175,11 +177,11 @@ def get_value_1_Bii
     end
 
     def get_value_1_Biii
-      open_cases_received_in_period.where("properties->>'external_deadline' < ?", @period_end).count
+      open_cases_received_in_period.where("properties->>'external_deadline' < ?", Date.today).count
```

Comment:
Can I just double check this change was intended? Seems curious on the face of it, since we use `@period_end` in other lines we've touched in this PR

Reply:
@Polsonby,   I really doubt they ever use this report because the basic scope of this report was wrong (it included all cases regardless which the type is relevant to this report or not) :)

if we use @period_end, it means the report only includes all those cases which haven't been closed up to last day of quarter. ---> which implies the cases being received nearly by end of the quarter will be excluded.

Bear in mind, the collection day for cabinet office report is 20 working day later then the last day of the quarter -> this requirement intends to include those open-date cases but being received at later stage of the quarter

Let me know, whether I have explained myself or not

Reply:
Yes, thanks for the clarification 👍 nice one